### ISSUE TITLE
VoiceOver: DOM tree, targets pane and content in 'styles tab' are defined as a table elements

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#131959]
[Check out Accessibility Insights](https://accessibilityinsights.io/en/downloads/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags**:#A11y_VSCode;#A11ySev2;#A11yTCS;#DesktopApp;#Visual Studio Code Client;#VoiceOver;#WCAG4.2.1;#DesktopApp;#Win10;#VisualStudioCode-Win32-Apr2021;#A11yMAS;

**Repro Steps:**

1. Launch Visual Studio code application with installed Elements for 'Microsoft edge extension .
2. Start screen reader
3. Navigate to 'targets' pane and observe voice over announcement.
Similarly navigate to DOM tree in developer tools and to the 'Styles' tab. 

**Actual Result:**
DOM is defined as table. When trying to interact with the table using (Ctrl+Option+Shift+Down arrow), Voiceover announces as 'Use navigation keys to interact.

**Expected Result:**
Elements in code editor field should not be defined as a table. They should be defined as list items since we are able to navigate to the elements using down arrow keys.

**User Impact:**
screen reader users will get the wrong information of content 
![Screenshot (47)](https://user-images.githubusercontent.com/24580570/131471478-cd87a936-4170-47c2-98dc-209c3780158c.png)
![Screenshot (48)](https://user-images.githubusercontent.com/24580570/131471494-8a2c1836-e25a-4c90-b119-d6c2e63bbf6b.png)

**Comment by isidorn** [comment]
If you take closer look at the Developer Tools you will see that the `role` of that widget is a `tree` which is expected here.
Also your developer tools screenshot does not really show anything invalid.
Thus closing as there is no change needed here, and nothing is broken for the user.
